tools/python/boutiques/neurolinks-template/tool.owl

```diff
@@ -0,0 +1,7 @@
+nl:neurodocker rdf:type owl:ObjectProperty ;
+    rdfs:label "@@__LABEL__@@"^^xsd:string ;
+    obo:IAO_0000115 "@@__DESCRIPTION__@@"^^xsd:string ;
+    prov:wasCreatedBy "@@__AUTHOR__@@"^^xsd:string ;
+    dct:source "@@__URL__@@"^^xsd:string ;
+    nl:has_boutiques:true			
```

Comment:
no semi-colon after this line, and I'm not sure about the `:` between `has_boutiques` and `true`, either... it didn't work at first attempt but I changed this and the above simultaneously so don't know what did the trick. I think this is a problem though.